instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/persistence_insert_class_methods.rb

```diff
@@ -3,6 +3,7 @@
 # Copyright The OpenTelemetry Authors
 #
 # SPDX-License-Identifier: Apache-2.0
+def ruby2_keywords(*) end if RUBY_VERSION < '2.7'
```

Comment:
One last small nit is we might want to protect "future us" and add a max RUBY_VERSION constraint of `>= 3.2` since the 3.2 release will likely coincide with `ruby2_keywords` getting removed, according to [ruby docs](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/) (h/t [dd-trace-rb](https://github.com/DataDog/dd-trace-rb/blob/4d77d37f7b94b845791c4695f6b75b9bef5f6256/lib/ddtrace/version.rb#L13-L23))
> If you really worry about the portability, use ruby2_keywords. (Acknowledge that Ruby 2.6 or before themselves have tons of corner cases in keyword arguments. :-) ruby2_keywords might be removed in the future after Ruby 2.6 reaches end-of-life. At that point, we recommend to explicitly delegate keyword arguments (see Ruby 3 code above).

```suggestion
def ruby2_keywords(*) end if RUBY_VERSION < '2.7' || RUBY_VERSION >= '3.2'
```

wdyt?

Reply:
https://github.com/ruby/ruby2_keywords should we use this gem? Its a more sureway to do the right thing here. I was first a little apprehensive to adding a dependency, wdyt?

Reply:
I think we can safely assume `ruby2_keywords` is just part of `ruby` itself as of 2.7, So i think as long as we protect ourselves on `<2.7 || >=3.2` versions, that's maybe a bit easier than adding a dependency? As 3.2 gets closer we can revisit, if they've kicked the can on deprecating it, is my two cents. Also our dependency min version might be differ from what an application has and could cause confusion or issues with installing this gem, so if we can avoid that i think why not?

- https://github.com/ruby/ruby/blob/82ea2870188d66aa75a99f03b4e7fdd1750aa196/vm_method.c#L2742
- https://github.com/ruby/ruby/commit/3b302ea8c95d34d5ef072d7e3b326f28a611e479